.idea/CodeBuddy.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Rather than create this separate .gitignore file, please ignore the entire .idea directory.